e2e/playwright/special-permissions-actions-available/src/tests/folders-actions.spec.ts

```diff
@@ -0,0 +1,134 @@
+
+    const initialFavoritesTotalItems = (await favoritesActions.getFavoritesTotalItems(username)) || 0;
+    await favoritesActions.addFavoritesByIds('folder', [folderFavId]);
+    await favoritesActions.addFavoritesByIds('file', [fileFavId]);
+    await favoritesActions.waitForApi(username, { expect: initialFavoritesTotalItems + 2 });
+  });
+
+  test.afterAll(async () => {
+    await nodesApi.deleteNodes([parentId]);
+    await trashCanActions.emptyTrashcan();
+  });
+
+  test.describe('on Personal Files : ', () => {
+    test('Folder not favorite  - [C213123]', async ({ personalFiles, loginPage }) => {
+      await personalFiles.navigate({ remoteUrl: `#/personal-files/${parentId}` });
+      await loginPage.loginUser({ username: username, password: username });
+      await checkActionsAvailable(personalFiles, testData.folderFile.name, testData.folderFile.toolbarPrimary, testData.folderFile.toolbarMore);
+      await personalFiles.reload();
+      await personalFiles.dataTable.getRowByName(testData.folderFile.name).click({ button: 'right' });
+      await personalFiles.matMenu.verifyActualMoreActions(testData.folderFile.contextMenu);
+    });
+
+    test('Folder favorite - [C280451]', async ({ personalFiles, loginPage }) => {
+      await personalFiles.navigate({ remoteUrl: `#/personal-files/${parentId}` });
+      await loginPage.loginUser({ username: username, password: username });
+      await checkActionsAvailable(
+        personalFiles,
+        testData.folderFavFile.name,
+        testData.folderFavFile.toolbarPrimary,
+        testData.folderFavFile.toolbarMore
+      );
+      await personalFiles.reload();
+      await personalFiles.dataTable.getRowByName(testData.folderFavFile.name).click({ button: 'right' });
+      await personalFiles.matMenu.verifyActualMoreActions(testData.folderFavFile.contextMenu);
+    });
+
+    test('multiple folders - [C280459]', async ({ personalFiles, loginPage }) => {
+      await personalFiles.navigate({ remoteUrl: `#/personal-files/${parentId}` });
+      await loginPage.loginUser({ username: username, password: username });
+      await personalFiles.dataTable.selectMultiItem(testData.folderFavFile.name, testData.folderFile.name);
+      await checkMultipleSelActionsAvailable(personalFiles, testData.multipleSelFile.toolbarPrimary, testData.multipleSelFile.toolbarMore);
+      await personalFiles.reload();
+      await personalFiles.dataTable.selectMultiItem(testData.folderFavFile.name, testData.folderFile.name);
+      await personalFiles.dataTable.getRowByName(testData.folderFavFile.name).click({ button: 'right' });
+      await personalFiles.matMenu.verifyActualMoreActions(testData.multipleSelFile.contextMenu);
+    });
+
+    test('both files and folders - [C280460]', async ({ personalFiles, loginPage }) => {
+      await personalFiles.navigate({ remoteUrl: `#/personal-files/${parentId}` });
+      await loginPage.loginUser({ username: username, password: username });
```

Comment:
same here, can be just { username, password: username }

Reply:
updated as per suggestion, thanks.